command/ca/acme/acme.go

```diff
@@ -0,0 +1,19 @@
+package acme
+
+import (
+	"github.com/smallstep/cli/command/ca/acme/eab"
+	"github.com/urfave/cli"
+)
+
+// Command returns the acme subcommand.
+func Command() cli.Command {
+	return cli.Command{
+		Name:        "acme",
+		Usage:       "manage ACME EAB",
+		UsageText:   "**step beta ca acme** <subcommand> [arguments] [global-flags] [subcommand-flags]",
+		Description: `**step beta ca acme** command group provides facilities for managing ACME EAB.`,
```

Comment:
The subcommand `acme` is for EAB related configuration, right? whereas, `acme` is more generally for ACME configuration. Or am I misunderstanding?

Reply:
Yes, that's the idea. There's only one subcommand under `acme` now though, which is why the usage now says `manage ACME EAB`. I'm not sure if there'll ever be more subcommands. Given that it's somewhat of a special kind of provisioner already, I don't think it's unlikely that there will be more subcommands in the future. WDYT?

Reply:
Let's keep the usage for ACME more general on the chance that we come back and add more stuff in here. There's a good chance we'd forget to update it if we do it the other way around.